src/migrations/1696628726163_user-notification-config-table.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+
+import Model from '../back/models/UserNotificationConfig'
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable(Model.tableName, {
+    address: {
+      primaryKey: true,
+      type: 'TEXT',
+      notNull: true,
+    },
+    last_notification_id: {
```

Comment:
It could be handy to have the created-at/updated_at for debug purposes.

For example, if at some point the service stop working or whatevs, we could know which where the last updated, or since when it stopped working, etc